firestore-counter/extension.yaml

```diff
@@ -36,13 +36,28 @@ contributors:
 roles:
   - role: datastore.user
     reason: Allows the extension to aggregate Cloud Firestore counter shards.
+  - role: pubsub.publisher
+    reason: Allows the HTTPS function to trigger the PubSub handler.
```

Comment:
1) PubSub -> Pub/Sub

2) We've not used the phrase "the Pub/Sub handler" anywhere else.  Should this be "... to trigger the Pub/Sub controllerCore function."?

Reply:
More specifically: "Allows the HTTPS controller function to publish a message to Pub/Sub, to trigger the controllerCore function"

Reply:
updated.  thx!